README.md

```diff
@@ -158,6 +158,7 @@ To learn more, see our [Documentation](https://skypilot.readthedocs.io/en/latest
 <!-- Keep this section in sync with index.rst in SkyPilot Docs -->
 Runnable examples:
```

Comment:
How about adding our blog and community integration page link to the sentence above as well?

Reply:
Done.